assets/example.tex

```diff
@@ -0,0 +1,216 @@
+% This example can be compiled using pdflatex if the SDAPS class is installed.
```

Comment:
Could you create an example that roughly resembles the competition sheets?
That is, 1 multiple choice fields for male/female (or something similar),
and then a lot of multiple choice fields with answer options A/B/C/D.

Do you know if the competition sheets will fit on one page? Do they include the question, or just the answer boxes?